### ISSUE TITLE
Function : gdml_train.train(task) failed for newest sgml version 0.4.10; 

### ISSUE THREAD

**Issue by a510396** [stefanch/sGDML#17]
Hi, It looks like gdml_train.train() would return a TypeError: the first argument must be callable

(It works well for previous version 0.4.4 sgml)
Just for newest sgml version 0.4.10; If we just run the demo code as below: 

import sys
import numpy as np
from sgdml.train import GDMLTrain

dataset = np.load('d_ethanol.npz')
n_train = 200

gdml_train = GDMLTrain()
task = gdml_train.create_task(dataset, n_train,\
        valid_dataset=dataset, n_valid=1000,\
        sig=10, lam=1e-15)

model = gdml_train.train(task)

File "/Users/sgdml/train.py", line 812, in train
    callback, disp_str='Generating descriptors and their Jacobians'; TypeError: the first argument must be callable

**Comment by AntonCh-G** [comment]
Dear a510396,
You should try to use ui from sgdml.utils as `callback=ui.callback`:
```
import sys
import numpy as np
from sgdml.train import GDMLTrain
from sgdml.utils import ui
import sys
import numpy as np
from sgdml.train import GDMLTrain

dataset = np.load('ethanol_dft.npz')
n_train = 200

gdml_train = GDMLTrain()
task = gdml_train.create_task(dataset, n_train,\
        valid_dataset=dataset, n_valid=1000,\
        sig=90, lam=1e-15,callback=ui.callback)

model = gdml_train.train(task,callback=ui.callback)

```

**Comment by stefanch** [comment]
Thank you for raising this issue! This bug has been fixed in sGDML 0.5.0.dev0 now.
Anton's workaround works great, in the meantime.
